recover_affected_workloads.md

```diff
@@ -61,3 +66,4 @@ TBD
 [1]: https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/
 [2]: https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-garbage-collection
 [3]: https://libvirt.org/formatdomain.html#watchdog-device
+[4]: https://kubernetes.io/docs/reference/labels-annotations-taints/#node-kubernetes-io-out-of-service
```

Comment:
I think their order should be switched because of my [above comment](https://github.com/medik8s/docs/pull/70/files#r1597584040)